librepo/downloader.c

```diff
@@ -293,7 +293,15 @@ is_max_mirrors_unlimited(const LrDownload *download)
 }
 
 static gboolean
-can_try_more(const LrDownload *download, int num_of_tried_mirrors, gboolean complete_path_or_baseurl)
+/**
+ * @brief Can be tried another mirror or retried baseurl or fullpath
```

Comment:
This description isn't great, how about:
> Returns whether the download can be retried, using the same URL in case of baseurl or full path, or using another mirror in case of using mirrors.